pages/docs/community-resources.mdx

```diff
@@ -49,34 +49,36 @@ YouTube (or blog)
 
 ## Contributions
 
-<table>
-  <thead>
-    <tr>
-      <th>CONTRIBUTOR</th>
-      <th>CONTRIBUTION</th>
-      <th>DATE (YYYY/MM/DD)</th>
-      <th>RESOURCE LINK (PR, Blog, Video)</th>
-    </tr>
-  </thead>
-  <tbody>
-    {data.map((item, index) => {
-      return (
-        <tr key={item.username + item.name + index}>
-          <td>
-            <a href={`https://github.com/${item.username}`}>{item.name}</a>
-          </td>
-          <td>{item.contribution}</td>
-          <td>{item.date}</td>
-          <td>
-            <a href={item.resource.url}>
-              {item.resource.name ? item.resource.name : item.resource.url}
-            </a>
-          </td>
-        </tr>
-      );
-    })}
-  </tbody>
-</table>
+<div className="w-full overflow-auto">
+  <table className="text-xs leading-5 md:text-base md:leading-none">
```

Comment:
you can remove all the classes here, we want to keep each page consistent in the way it is styled and thus far we do not include any inline styling at the table level on any docs pages. Otherwise the changes look good! 
```suggestion
  <table>
```